tests/pytests/unit/modules/test_rpm_lowpkg.py

```diff
@@ -281,6 +281,8 @@ def test_version_cmp_rpm_all_libraries(rpm_lib):
         assert 0 == rpm.version_cmp("3:2.9.1-6.el7.4", "3:2.9.1-6.el7.4")
         assert -1 == rpm.version_cmp("3:2.9.1-6.el7.4", "3:2.9.1-7.el7.4")
         assert 1 == rpm.version_cmp("3:2.9.1-8.el7.4", "3:2.9.1-7.el7.4")
+        assert 0 == rpm.version_cmp("3.23-6.el9", "3.23")
```

Comment:
```suggestion
        rpm.version_cmp("3.23-6.el9", "3.23") == 0
```
Let's take the opportunity to fix the assertions. What we expect should be on the right.

Reply:
The reason being that when it fails, the pytest report will make more sense. This is specially important with lists and dictionaries, not this case, but at least someone looking will know how it should be done.